static/templates/admin_linkifier_edit_form.hbs

```diff
@@ -0,0 +1,27 @@
+<div id="linkifier-info-form-modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="linkifier-info-form-modal-label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="linkifier-info-form-modal-label">{{t "Edit linkifiers" }}</h3>
+        <div class="alert-notification edit-filter-status"></div>
+    </div>
+    <div class="modal-body">
+        <div id="linkifier-form" data-filter-id="{{filter_id}}">
```

Comment:
Might as well call this `data-linkifier-id` just to avoid a migration later.

Reply:
Done.